app/main.py

```diff
@@ -41,7 +41,7 @@ def create_tables(engine, psql_environment):
 
     agenda, calendar, categories, celebrity, currency, dayview,
     email, event, export, four_o_four, invitation, login, logout, profile,
-    register, search, telegram, weekview, whatsapp,
+    register, search, telegram, weekview, whatsapp, google_connect
```

Comment:
Put it in the right place alphabetically

Reply:
Done.